build/ProcessJarFiles/CommonProperties.props

```diff
@@ -20,7 +20,7 @@
     <!-- Jar used to extract rule metadata from a plugin file, created for us by the SL Java folk.
          For simplicity, the file is checked in under build\tools\binaries. -->
     <!-- Obtained from https://repox.jfrog.io/ui/native/sonarsource/org/sonarsource/sonarlint/core/sonarlint-rule-extractor-cli/8.13.0.60370/ -->
-    <RuleJsonExtractorJar>$(MSBuildThisFileDirectory)..\tools\binaries\sonarlint-rule-extractor-cli-8.13.0.60370-jar-with-dependencies.jar</RuleJsonExtractorJar>
+    <RuleJsonExtractorJar>$(MSBuildThisFileDirectory)..\tools\binaries\sonarlint-rule-extractor-cli-9.2.0.74440-jar-with-dependencies.jar</RuleJsonExtractorJar>
```

Comment:
That's the stable release, right?

Reply:
it's not the stable release. As we talked in the morning because it's an internal tool we can use it as long as it works.